docs/rnaseq-rmd.rst

```diff
@@ -260,6 +260,23 @@ adds some extra convenience when working with lists of dds objects, including
 the detection of parallelization as set up in the config object. See the help
 for ``lcdbwf::make_results()`` for more details.
 
+By default, if no test argument is specified in the parameters for
+``lcdbwf::make_dds`` (examples 1-4, rnaseq.Rmd, lines 164-187), the Wald test is
+performed. When ``lcdbwf::make_results`` processes a Wald test dds object, it
+detects the Wald test and expects a ``contrast`` or ``coef`` argument to specify which
+p-values and log2FoldChange values to report.
+
+DESeq2 also supports the nBinomLRT (LRT). Example 5 (rnaseq.Rmd, line 189)
+demonstrates how to create a dds object with LRT data. Since the LRT tests
+the removal of one or more terms from the design formula, a single
+log2FoldChange column doesn't reflect the test's complexity. DESeq2's results
+object is optimized for the Wald test, and when storing LRT results, it
+maintains consistency in datastructure by choosing a single pair-wise comparison for
+log2FoldChange values. To avoid confusion, we set all log2FoldChange values to
+0 for LRT results.
```

Comment:
This is very unexpected behavior, so should be more prominent in the docs

```suggestion
log2FoldChange values. To avoid confusion, ***we set all log2FoldChange values to
0 for LRT results***.
```